### PR TITLE
update example/rethinkdb to v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
update example/rethinkdb to v1beta3, add external load balancer to work on GCE, add a comment about nodeSelector in gen-pod.sh.
@nikhiljindal @antmanler

### CODE REVIEW

**Comment by caesarxuchao** [comment]
@nikhiljindal The UI works. But I think I need to change the run.sh as well, as pointed out in #7258. Please hold until I fix it. Thanks.

**Comment by bgrant0607** [comment]
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/rethinkdb/image/run.sh#L26

**Comment by caesarxuchao** [comment]
I fix the run.sh to v1beta3. @antmanler, could you rebuild your docker image? or you can copy mine at caesarxuchao/rethinkdb:v2. Thank you.

@nikhiljindal  let's wait for greens

File: examples/rethinkdb/README.md [hunk 1]
```diff
@@ -81,12 +91,28 @@ find the service
...
+```
+createExternalLoadBalancer: true
 ```
 
-open a web browser and access to *http://10.241.220.209:8080* to manage you cluster
+The external load balancer allows us to access the service from outside via an external IP, which is 104.197.19.120 in this case. 
+
+Note that you may need to create a firewall rule to allow the traffic, assuming you are using GCE:
+```
+$ gcloud compute firewall-rules create rehitnkdb --allow=tcp:8080
```
Comment (nikhiljindal) [inline examples/rethinkdb/README.md]: typo in rethinkdb

File: examples/rethinkdb/gen-pod.sh [hunk 2]
```diff
@@ -32,41 +32,42 @@ if [[ ${NAME} == "admin" ]]; then
 fi
 
 NODE=""
+# One needs to label a node with the same key/value pair, 
+# i.e., 'kubectl lable nodes <node-name> name=${2}'
```
Comment (nikhiljindal) [inline examples/rethinkdb/gen-pod.sh]: lable -> label

File: examples/rethinkdb/README.md [hunk 3]
```diff
@@ -10,6 +10,12 @@ Setting up a [rethinkdb](http://rethinkdb.com/) cluster on [kubernetes](http://k
 
 Quick start
 -----------
+**Step 0**
```
Comment (nikhiljindal) [inline examples/rethinkdb/README.md]: Why is this required?

**Comment by nikhiljindal** [comment]
Will wait for @antmanler's LGTM before merging.

File: examples/rethinkdb/README.md (hunk 3 above)
Comment (caesarxuchao) [inline examples/rethinkdb/README.md]: The author of this example made everything under rethinkdb namespace..

**Comment by caesarxuchao** [comment]
@nikhiljindal. Thank you for the picks. I was too hurry to go home : )

**Comment by antmanler** [comment]
@caesarxuchao Thank you for you contribution, I'm so busy these days, and sorry for late reply.

I've copied and pushed your image to antmanler/rethinkdb, and it works well on my local cluster.

LGTM.

File: examples/rethinkdb/README.md [hunk 4]
```diff
@@ -10,6 +10,12 @@ Setting up a [rethinkdb](http://rethinkdb.com/) cluster on [kubernetes](http://k
...
+
+change the namespace of the current context to "rethinkdb"
+```
+$kubectl config view | grep "current-context" | awk '{print $2}' | xargs -I {} kubectl config set-context {} --namespace=rethinkdb
```
Comment (nikhiljindal) [inline examples/rethinkdb/README.md]: We use 
```config view -o template --template='{{index . "current-context"}}'``` at other places like https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/kubectl.sh#L127 which seems cleaner.

Comment (caesarxuchao) [inline examples/rethinkdb/README.md]: Updated.

**Comment by nikhiljindal** [comment]
LGTM, thanks.
Shippable is green, merging.
